### PR TITLE
add switcher in job manager to switch launcher

### PR DESCRIPTION
**Pull request by xudifsd**
Will use `dlws/frameworkcontroller` as controller image, because upstream do not have [fix](https://github.com/microsoft/frameworkcontroller/pull/53#issuecomment-588008410) we needed. Will change to upstream image once it's released.

### CODE REVIEW

File: src/ClusterBootstrap/services/launcher/launcher.yaml [hunk 1]
```diff
@@ -19,4 +19,4 @@ spec:
       # serviceAccountName: frameworkcontroller
       containers:
       - name: frameworkcontroller
-        image: frameworkcontroller/frameworkcontroller:v0.5.1
+        image: xudifsd/frameworkcontroller
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/services/launcher/launcher.yaml]: We probably should not use personal identifier in repo?

File: src/docker-images/RestfulAPI/runScheduler.sh [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
-rm /DLWorkspace/src/utils/config.yaml
```
Comment (Anbang-Hu) [inline src/docker-images/RestfulAPI/runScheduler.sh]: Does `ln` replace the old target?

File: src/ClusterBootstrap/services/launcher/launcher.yaml (hunk 1 above)
Comment (xudifsd) [reply src/ClusterBootstrap/services/launcher/launcher.yaml]: Yes.. I also think so.

File: src/docker-images/RestfulAPI/runScheduler.sh (hunk 2 above)
Comment (xudifsd) [reply src/docker-images/RestfulAPI/runScheduler.sh]: No, there are no such file in image, I think this is used for previous deployment when they tried to debug with local change mount directly into this directory. But we don't do this anymore.

**Pull request merged**